capstone/src/main/webapp/youtubeTypescript.js

```diff
@@ -577,3 +583,18 @@ function drawTableChart() {
     }
     table.draw(tableData, { allowHtml: true, showRowNumber: false, width: '100%', height: '100%' });
 }
+/** Displays a loading wheel that can be used a placeholder until an output is ready to be displayed*/
+function showLoadingWheel() {
+    var loadingContainerElement = document.getElementById('loading-container');
+    // Only one loading wheel will be shown at a time
+    if (loadingContainerElement.innerHTML == '') {
+        var loadingWheel = document.createElement('p');
```

Comment:
nit: Let's use `const`s or `let`s where possible.

Reply:
I have added youtubeTypescript.js to .gitignore in "add gitignore and delete cache"